packages/office-addin-lint/src/lint.ts

```diff
@@ -26,8 +31,14 @@ export function getLintCheckCommand(files: string): string {
 }
 
 export function performLintCheck(files: string) {
-  const command = getLintCheckCommand(files);
-  execCommand(command);
+  try {
+    const command = getLintCheckCommand(files);
+    execCommand(command);
+    usageDataHelper.sendUsageDataSuccessEvent("check");
+  } catch (err) {
+    usageDataHelper.sendUsageDataException("check", `${err}`);
```

Comment:
I'm not so sure about the conversion of `err` to a string here. `err.message` would be the message whereas this would stringify the whole object. Also, you cannot be sure of the type for `err` which could be an Error object, a string, or more rarely something else. There is a function in the packages to do this check. I need to look it up, but using a helper function to convert `err` to a string would be recommended.